build_tools/gradle/build_tflite_android_library.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Cross-compile IREE's TFLite Java Bindings with Gradle and CMake. Produces an
+# iree-tflite-bindings-debug.aar and an iree-tflite-bindings-release.aar
+# library under bindings/tflite/java/output. Designed for CI, but can be run
+# manually from within the https://github.com/google/iree repo.
+
+set -x
+set -e
+set -o pipefail
+
+# --------------------------------------------------------------------------- #
+ROOT_DIR=$(git rev-parse --show-toplevel)
+
+CMAKE_BIN=${CMAKE_BIN:-$(which cmake)}
+
+"${CMAKE_BIN?}" --version
+ninja --version
+gradle --version
+
+cd ${ROOT_DIR?}
+
+# --------------------------------------------------------------------------- #
+# Build the host libraries
+
+HOST_BUILD_DIR=${ROOT_DIR?}/build-host
+HOST_INSTALL_DIR=${HOST_BUILD_DIR}/install
+
+cmake -G Ninja -B ${HOST_BUILD_DIR} \
+-DIREE_BUILD_COMPILER=OFF \
+-DIREE_BUILD_TESTS=OFF \
+-DIREE_BUILD_SAMPLES=OFF \
+-DCMAKE_C_COMPILER=clang \
+-DCMAKE_CXX_COMPILER=clang++ \
+-DCMAKE_INSTALL_PREFIX=${HOST_INSTALL_DIR} .
```

Comment:
nit: indent when splitting a command over multiple lines

```suggestion
cmake -G Ninja -B ${HOST_BUILD_DIR} \
  -DIREE_BUILD_COMPILER=OFF \
  -DIREE_BUILD_TESTS=OFF \
  -DIREE_BUILD_SAMPLES=OFF \
  -DCMAKE_C_COMPILER=clang \
  -DCMAKE_CXX_COMPILER=clang++ \
  -DCMAKE_INSTALL_PREFIX=${HOST_INSTALL_DIR} .
```